Make.inc

```diff
@@ -273,7 +273,7 @@ endif
 STD_LIB_PATH ?= $(PATH)
 
 JLDOWNLOAD := $(JULIAHOME)/deps/tools/jldownload
-JLCHECKSUM := $(JULIAHOME)/deps/tools/jlchecksum
+JLCHECKSUM := $(JULIAHOME)/deps/tools/jlchecksum.sh
```

Comment:
So from quick look at the filename someone knows it's a shell script without needing to open it.
And because besides 3 scripts, all others had .sh extension. So why not all?